Source/Urho3D/Container/Vector.h

```diff
@@ -192,24 +213,61 @@ template <class T> class Vector : public VectorBase
         return Buffer()[index];
     }
 
+    /// Create an element at the end.
+    template <class... Args> T& Emplace(Args&&... args)
+    {
+        if (size_ < capacity_)
+        {
+            // Optimize common case
+            ++size_;
+            new (&Back()) T(std::forward<Args>(args)...);
+        }
+        else
+        {
+            T value(std::forward<Args>(args)...);
+            Push(std::move(value));
+        }
+        return Back();
+    }
+

```

Comment:
Standard C++ library provides insert/push_back, emplace/emplace_back.

I see Emplace is same as STL emplace_back.

A little confused.

Then where is STL emplace?

Reply:
Ok, I thought about naming it `EmplaceBack`.
Since you mentioned it too, I'll rename this function.
About `emplace`... I don't consider this function so useful and I'm too lazy to implement it, at least in this PR.
It'd be just construct & insert anyway.
BTW, &&-inserts are missing too.
Next time, maybe.